### ISSUE TITLE
Add option to read settings from a config file

### ISSUE THREAD

**Issue by rach** [rach/pome#41]

**Comment by rach** [comment]
I do have a preference for toml but it's maybe not a well knows format

**Comment by rlayte** [comment]
kingpin accepts values from a file, but in a non standard format. Might be the easiest option for now though.

https://godoc.org/github.com/alecthomas/kingpin#ExpandArgsFromFile

**Comment by rach** [comment]
Agree. Better to have temporary easy solution for now as the flags are going to grow and then deciding on the best suited format. Thanks for find this.
